### PR TITLE
Use gtk_builder_new_from_resource

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: capplets/appearance/appearance-main.c [hunk 1]
```diff
@@ -44,34 +44,24 @@ init_appearance_data (int *argc, char ***argv, GOptionContext *context)
   activate_settings_daemon ();
 
   /* set up the data */
-  ui = gtk_builder_new ();
-  gtk_builder_add_from_resource (ui, "/org/mate/mcc/appearance/data/appearance.ui", &err);
-
-  if (err)
```
Comment (zhuyaliang) [inline capplets/appearance/appearance-main.c:50]: Remove this useless variable ```GError *err = NULL```

File: capplets/default-applications/mate-da-capplet.c [hunk 2]
```diff
@@ -588,21 +588,7 @@ show_dialog(MateDACapplet* capplet, const gchar* start_page)
...
-		GtkWidget* dialog = gtk_message_dialog_new (NULL, 0, GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, _("Could not load the main interface"));
-		gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog), _("Please make sure that the applet is properly installed"));
-		gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_OK);
-
-		gtk_dialog_run(GTK_DIALOG(dialog));
-
-		gtk_widget_destroy(dialog);
-		exit(EXIT_FAILURE);
-	}
+	capplet->builder = builder = gtk_builder_new_from_resource ("/org/mate/mcc/da/mate-default-applications-properties.ui");
```
Comment (zhuyaliang) [inline capplets/default-applications/mate-da-capplet.c:591]: I'm not sure if I should add ```g_object_unref (builder);```

**Comment by zhuyaliang** [comment]
@rbuj 5 commits should be compressed into one

File: capplets/appearance/appearance-main.c (hunk 1 above)
Comment (rbuj) [reply capplets/appearance/appearance-main.c:50]:
```
appearance-main.c:40:11: warning: unused variable 'err' [-Wunused-variable]
   40 |   GError *err = NULL;
      |           ^~~
```

File: capplets/default-applications/mate-da-capplet.c [hunk 3]
```diff
@@ -588,21 +588,7 @@ show_dialog(MateDACapplet* capplet, const gchar* start_page)
 	GtkBuilder* builder;
 	guint builder_result;
```
[the remark this answers is not in the document]
Comment (rbuj) [reply capplets/default-applications/mate-da-capplet.c:589]:
```
mate-da-capplet.c:589:8: warning: unused variable 'builder_result' [-Wunused-variable]
  589 |  guint builder_result;
      |        ^~~~~~~~~~~~~~
```

File: capplets/default-applications/mate-da-capplet.c (hunk 2 above)
Comment (rbuj) [reply capplets/default-applications/mate-da-capplet.c:590]: Where did you decrease the reference count of builder object?

**Comment by rbuj** [comment]
> @rbuj 5 commits should be compressed into one

No, I think it's fine as it's now.
However, if there is someone else who think that all the commits should be squashed in a single one, then I'll do it.

**Review by raveit65 — APPROVED** [review]
LGTM,
builds and runs fine.

**Comment by raveit65** [comment]
Any open question? LGTM to use single commits.

**Review by zhuyaliang — APPROVED** [review]

**Pull request merged**